argobots.spec

```
@@ -1,18 +1,22 @@
 Name: argobots
 
 %global major 1
-%global minor 1
+%global minor 2
 #%%global prerelease
 %global tag %{major}.%{minor}%{?prerelease}
 
 Version: %{major}.%{minor}%{?prerelease:~%{prerelease}}
-Release: 3%{?dist}
+Release: 1%{?dist}
 Summary: Lightweight, low-level threading and tasking framework
 Group: System Environment/Libraries
 License: UChicago Argonne, LLC -- Argobots License
 Url: http://www.argobots.org/
-#Source: https://api.github.com/repos/pmodels/$(NAME)/tarball/31703b1
 Source: https://github.com/pmodels/%{name}/releases/download/v%{tag}/%{name}-%{tag}.tar.gz
+%if "%{?commit}" != ""
+Patch0: %{version}..%{commit}.patch
+%endif
+# fix ULTs stacks dump works only once
+Patch1: https://github.com/pmodels/argobots/commit/411e5b344642ebc82190fd8b125db512e5b449d1
```

Oops.  My previous suggestion was not quite right.  This needs to be:

```suggestion
Patch1: https://github.com/pmodels/argobots/commit/411e5b344642ebc82190fd8b125db512e5b449d1.patch
```

since we want the raw patch, not the prettified GitHub display of this commit.  This will also fix the failure in Jenkins.

Apologies for the previous error on my part.

---

I also plaid guilty to not have check it...

Fixed with commit ad39283